hq_superset/models.py

```diff
@@ -34,6 +35,10 @@ class DataSetChange:
     data: list[dict[str, Any]]
 
     def update_dataset(self):
+        with statsd.timed('cca.dataset_change.timer', tags=[self.data_source_id]):
```

Comment:
:fire:

Reply:
and :moneybag: ?

Reply:
I'm currently busy figuring that out, since this would potentially result in massive volumes.